components/camera/videosource/webcam.go

```diff
@@ -208,43 +208,16 @@ func NewWebcamSource(ctx context.Context, attrs *WebcamAttrs, logger golog.Logge
 		return tryWebcamOpen(ctx, attrs, attrs.Path, false, constraints, logger)
 	}
 
-	var pattern *regexp.Regexp
-	if attrs.PathPattern != "" {
```

Comment:
this deletes path pattern which we still need

Reply:
Unclear that we do... according to Fahmina, according to the consulting team, zero people are actually using it (see comments: https://docs.google.com/document/d/18p3LV6ghH-m0xGjO7MZRz4zK0X8JePmGZRAXBtm0k7E/edit) and there's not much reason to compare a regex pattern anymore

Reply:
Once the scope doc is approved, it will remove PathPattern, and this will be OK.

Reply:
I didn't see this scope